vars/withDefinitionImportSecretsAndEnvVars.groovy

```diff
@@ -0,0 +1,31 @@
+
+def call(String product, String environment, Map<String, String> vaultOverrides, Closure block) {
+  def dependedEnv = vaultOverrides.get(environment, environment)
+
+  env.IDAM_URL = "https://idam-api.${dependedEnv}.platform.hmcts.net"
```

Comment:
this needs to be overridden in prod block too as its idam-api.platform.hmcts.net